<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibratorWidget</class>
 <widget class="QWidget" name="CalibratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbAction">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Шаг, мм</string>
   </property>
   <widget class="QRadioButton" name="rbX01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&amp;x 0.1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbX10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>x &amp;1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="videoLayout"/>
  </widget>
  <widget class="QLabel" name="lbHoleName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>35</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Hole:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHoleInfo">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>10</y>
     <width>24</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbTempInfo">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>28</y>
     <width>42</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNext">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
